DigitalOffice/Tests/ProjectServiceTests/CommandsTests/GetProjectInfoByIdCommandTests.cs

```diff
@@ -0,0 +1,102 @@
+using Moq;
+using NUnit.Framework;
+using LT.DigitalOffice.ProjectService.Commands;
+using LT.DigitalOffice.ProjectService.Commands.Interfaces;
+using LT.DigitalOffice.ProjectService.Database.Entities;
+using LT.DigitalOffice.ProjectService.Mappers;
+using LT.DigitalOffice.ProjectService.Mappers.Interfaces;
+using LT.DigitalOffice.ProjectService.Models;
+using LT.DigitalOffice.ProjectService.Repositories.Interfaces;
+using LT.DigitalOffice.ProjectServiceUnitTests.UnitTestLibrary;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace LT.DigitalOffice.ProjectServiceUnitTests.CommandsTests
+{
+    public class GetProjectInfoByIdCommandTests
+    {
+        private IGetProjectInfoByIdCommand command;
+        private Mock<IProjectRepository> repositoryMock;
+        private Mock<IMapper<DbProject, Project>> mapperMock;
+
+        private DbProjectManagerUser dbManagersIds;
+        private DbProjectWorkerUser dbWorkersIds;
+        private DbProject project;
+
+        private Guid projectId;
+        private Guid workerId;
+        private Guid managerId;
+
+        private string Name = "Project";
+
+        [SetUp]
+        public void Setup()
+        {
+            repositoryMock = new Mock<IProjectRepository>();
+            mapperMock = new Mock<IMapper<DbProject, Project>>();
+            command = new GetProjectInfoByIdCommand(repositoryMock.Object, mapperMock.Object);
+
+            workerId = Guid.NewGuid();
+            managerId = Guid.NewGuid();
+            projectId = Guid.NewGuid();
+            dbManagersIds = new DbProjectManagerUser
+            {
+                ProjectId = projectId,
+                Project = project,
+                ManagerUserId = managerId
+            };
+            dbWorkersIds = new DbProjectWorkerUser
+            {
+                ProjectId = projectId,
+                Project = project,
+                WorkerUserId = workerId
+            };
+            project = new DbProject
+            {
+                Id = projectId,
+                Name = Name,
+                WorkersUsersIds = new List<DbProjectWorkerUser> { dbWorkersIds },
+                ManagersUsersIds = new List<DbProjectManagerUser> { dbManagersIds }
+            };
+        }
+
+        [Test]
+        public void ThrowsExceptionIfRepositoryThrowsIt()
+        {
```

Comment:
At the end of the test name should be **Test** (convention), do it to the all test.

Reply:
That's only for classes and project names.
![image](https://user-images.githubusercontent.com/59228285/89103276-47143900-d419-11ea-973d-29856e20c86f.png)

Reply:
And it has been already done :)